__docs__/foundations-color.mdx

```diff
@@ -88,27 +89,15 @@ would use `core.border.neutral.subtle`, `core.border.neutral.default` for when
 3:1 contrast is a priority (ex. form elements) and `core.border.neutral.strong`
 for 4.5:1 contrast.
 
-<View style={styles.gridCompact}>
-    <ActionColorGroup
```

Comment:
I found that the React components were not properly reactive when embedded in a `.mdx` file 🥲  I moved the jsx to a story and reference that story using a `Canvas`. 

Wanted to get more feedback on this approach before doing the same for the rest of the semantic color tokens.

I think the token tables will also need something similar so that the values can be updated when the theme changes

Reply:
This makes total sense! thanks for improving it :)